spec/facades/user_campaigns_with_characters_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+require_relative '../../app/facades/user_campaigns_with_characters'
+
+describe 'UserCampaignsWithCharacters' do
+
+  it 'Should have respond to user' do
```

Comment:
This test does not add a lot of value, if you remove the implementation of the code, your tests will still pass....